README.md

```diff
@@ -0,0 +1,165 @@
+#  ![Butterfly Icon](distribute/bficon.jpg) Butterfly
+
+    File: Butterfly-v1.6.pkg
+    Author: Thomas THERY <dragfly_kj@rocketmail.com>
+    Release: v1.6.1 (Nov 2001)
+    Compatibility: BeOS R5 for Intel
+    Description: Picture browser
+    Notes: Freeware
+
+Butterfly is a picture browser based on the translators for displaying bitmap images. It was inspired by Sea, a DOS picture browser. Its aim is to allow easy and fast browsing of an important number of pictures.
```

Comment:
"an important number"?  Maybe "a large number"?

Reply:
I did not edit anything, all of these lines are the author's original.